libraries/libcore/test/test_status.c

```diff
@@ -59,13 +59,29 @@ static StatusCode prv_ok_or_return() {
   return status_code(STATUS_CODE_UNKNOWN);
 }
 
+// Increments an external count to check how many times the function is called
+static StatusCode prv_status_num_calls(uint16_t *count) {
+  (*count)++;
+  return STATUS_CODE_OK;
```

Comment:
Since this function returns `STATUS_CODE_OK`, this will actually pass with the old `status_ok_or_return` function - have this return some non-ok code just to make sure it actually tests the thing.